Makefile

```diff
@@ -4,7 +4,7 @@ VERSION = 0.9.11
 
 PREFIX = /usr/local
 INSTALL_PATH = $(PREFIX)/bin/$(EXECUTABLE_NAME)
-BUILD_PATH = .build/apple/Products/Release/$(EXECUTABLE_NAME)
+BUILD_PATH = .build/release/$(EXECUTABLE_NAME)
```

Comment:
Can you confirm that this doesn't break the Mac build?

Reply:
Actually it doesn't, fixed now